integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -248,6 +248,21 @@ const stubVerifyExpiredBarcode = (): SuperAgentRequest =>
     },
   })
 
+const stubVerifyNotFoundBarcode = (): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'POST',
+      urlPattern: '/send-legal-mail/barcode/check',
+      bodyPatterns: [{ matchesJsonPath: '$[?(@.barcode == "666656789012")]' }],
```

Comment:
I wonder if we could create a bunch of constants for these barcodes with descriptive names to indicate what each of them does?

Reply:
Good plan. Yes, I'll do that in a 'tidy things up' type PR (there are a couple of things I want to tidy)